Classes/Domain/Repository/GlossaryEntryRepository.php

```diff
@@ -48,6 +48,26 @@ public function findByLanguages(array $languages): array
         return $query->execute()->toArray();
     }
 
+    /**
+     * @param string $text
```

Comment:
Does not add value to the PHP Type, so could be omitted. (And describes only 1 of 2 arguments.)

Reply:
Removed the `@param` notation 👍🏻.